lib/utils.js

```diff
@@ -371,13 +371,29 @@ function saveAndReplaceDotEnvCredentials (dotenvFile, saveFile, apihost, namespa
   fs.moveSync(dotenvFile, saveFile)
   // Only override needed env vars and preserve other vars in .env
   const env = dotenv.parse(fs.readFileSync(saveFile))
-  env.AIO_RUNTIME_APIHOST = apihost
-  env.AIO_RUNTIME_AUTH = auth
-  env.AIO_RUNTIME_NAMESPACE = namespace
-  // existing AIO__ vars might override above AIO_ vars
-  delete env.AIO__RUNTIME_AUTH
```

Comment:
we should still handle double underscores as config lib treats `AIO_` and `AIO__` in the same way

Reply:
👍  because of this quirk: https://github.com/adobe/aio-lib-core-config/issues/49